### PR TITLE
Remove old `setWindowTitle` global function

### PR DESCRIPTION
**Pull request by sbc100**
Instead just move the single implementation to
`emscripten_set_window_title` alongside `emscripten_get_window_title`.

### CODE REVIEW

File: test/other/metadce/test_metadce_hello_O0.jssize [hunk 1]
```diff
@@ -1 +1 @@
-23664
+23677
```
Comment (sbc100) [inline test/other/metadce/test_metadce_hello_O0.jssize:1]: This increase is just due to the longer `assert` message

File: src/library_glfw.js [hunk 2]
```diff
@@ -636,9 +636,9 @@ var LibraryGLFW = {
       var win = GLFW.WindowFromId(winid);
       if (!win) return;
 
-      win.title = UTF8ToString(title);
+      win.title = title;
       if (GLFW.active.id == win.id) {
-        document.title = win.title;
+        emscripten_set_window_title(title);
```
Comment (kripken) [inline src/library_glfw.js:641]: Doesn't this need `_` prefix?

Comment (sbc100) [reply src/library_glfw.js:641]: Oops yes, good eye

File: src/library_browser.js [hunk 3]
```diff
@@ -1120,9 +1120,7 @@ var LibraryBrowser = {
   },
 
   emscripten_set_window_title__proxy: 'sync',
-  emscripten_set_window_title: (title) => {
-    setWindowTitle(UTF8ToString(title));
-  },
+  emscripten_set_window_title: (title) => document.title = UTF8ToString(title),
```
Comment (kripken) [inline src/library_browser.js:1123]: Do we need a dep on `UTF8ToString` here?

**Review by kripken — COMMENTED** [review]
lgtm % comment

File: src/library_browser.js (hunk 3 above)
Comment (sbc100) [reply src/library_browser.js:1123]: That missing dep is pre-existing, so this change should effect that.

I just checked and it looks like the reason this is OK is that we have `UTF8ToString` listed in the implicit deps list: https://github.com/emscripten-core/emscripten/blob/dfb3585e8f337cb010c7ed09a3942d436633217c/src/jsifier.js#L270-L276. 

So think we can leave it as is for now.

**Review by kripken — APPROVED** [review]

**Pull request merged**